src/main/java/amazon/SearchResultPage.java

```diff
@@ -69,4 +71,12 @@ public void waitForPageLoad() {
                 until(ExpectedConditions.elementToBeClickable(countOfItem.get(countOfItemInPage() - 1)));
     }
 
+    public void seeAllResult() {
+        seeAllSearchResultButton.click();
+    }
+
+    public void waitForPageLoadALlSearchButton() {
+        new WebDriverWait(driver, Duration.ofSeconds(20)).until(ExpectedConditions.visibilityOf(searchResultName));
```

Comment:
This wait should be for the whole page. It might contain more than 1 waiting conditions, that's why this method name should be "waitForPageLoad"